test/acceptance/top-contributors-test.js

```diff
@@ -87,3 +87,75 @@ acceptance("Right Sidebar - Top Contributors - Custom count", function (needs) {
     );
   });
 });
+
+acceptance(
+  "Right Sidebar - Top Contributors - Custom order, group exclusion and period",
+  function (needs) {
+    const blocksJSON = [
+      {
+        name: "top-contributors",
+        params: [
+          {
+            name: "count",
+            value: "3",
+          },
+          {
+            name: "order",
+            value: "likes_given",
+          },
+          {
+            name: "period",
+            value: "weekly",
+          },
+          {
+            name: "excludedGroupNames",
+            value: "Team|contributors",
+          },
+        ],
+      },
+    ];
+
+    needs.hooks.beforeEach(() => {
+      settings.blocks = JSON.stringify(blocksJSON);
+    });
+
+    needs.hooks.afterEach(() => {
+      settings.blocks = "[]";
+    });
+
+    needs.pretender((server, helper) => {
+      server.get("/directory_items.json", (request) => {
+        if (
+          request.queryParams.period === "weekly" &&
+          request.queryParams.order === "likes_given" &&
+          request.queryParams.exclude_groups === "Team|contributors" &&
+          request.queryParams.limit === "3"
+        ) {
+          return helper.response(DirectoryFixtures["directory_items"]);
+        }
+
+        return helper.response(400, {}, { error: "Bad Request" });
+      });
+    });
+
+    test("Viewing latest", async function (assert) {
+      await visit("/");
+
+      assert.ok(visible(".tc-right-sidebar"), "sidebar element is present");
+      assert.ok(
+        visible(".top-contributors--container"),
+        "top contributors element is present"
+      );
+
+      assert.strictEqual(
+        queryAll(".top-contributors--user").length,
+        3,
+        "custom limit is respected"
+      );
```

Comment:
The other assertions could use the qunit-dom goodness too 😃 

```suggestion
      assert.dom(".tc-right-sidebar").isVisible("sidebar element is present");
      assert
        .dom(".top-contributors--container")
        .isVisible("top contributors element is present");

      assert
        .dom(".top-contributors--user")
        .exists({ count: 3 }, "custom limit is respected");
```